app/models/concerns/legal_aid_application_state_machine.rb

```diff
@@ -127,6 +130,10 @@ def provider_checking_or_checked_citizens_means_answers?
       event :submitted_assessment do
         transitions from: :submitting_assessment, to: :assessment_submitted
       end
+
+      event :reset_from_use_ccms do
+        transitions from: :use_ccms, to: :initiated
```

Comment:
My only concern is around this bit at the moment...
We transition _to_ use_ccms from a number of states, see line 68 above for another example.
Block reversion to initiated may cause issues with other states.

Reply:
No, that's cool :+1: